dcpy/connectors/edm/recipes.py

```diff
@@ -248,7 +256,10 @@ def import_datasets(planned_recipe_file: Path = _typer_recipe_file_opt):
     """
     logger.info("Importing Recipe Datasets")
     recipe = recipe_from_yaml(planned_recipe_file)
-    [import_dataset(ds) for ds in recipe.inputs.datasets]
+    pg_client = postgres.PostgresClient(
+        schema=git.run_name(),
```

Comment:
@damonmcc Let's extract this to be an env var

Reply:
see [be9a76f](https://github.com/NYCPlanning/data-engineering/pull/234/commits/be9a76f1b4995afb4ea58d15b68baaf2a89e2e8b)

was tempting to add an argument to `import_datasets()` but that felt like a violation of it's intent as a CLI command